content/influxdb/v2.1/security/tokens/create-token.md

```diff
@@ -15,26 +15,90 @@ Create API tokens using the InfluxDB user interface (UI), the `influx`
+
+{{% oss-only %}}Tokens are visible to the user who created the token. Users who own a token with Operator permissions also have access to all tokens.
 Tokens stop working when the user who created the token is deleted.
+
 **We recommend creating a generic user to create and manage tokens for writing data.**
+{{% /oss-only %}}
+
+{{% cloud-only %}}
+To follow best practices for secure API token generation and retrieval, InfluxDB Cloud enforces access restrictions on API tokens.
+  - Tokens are visible to the user who created the token.
+  - InfluxDB Cloud UI only allows access to the API token value immediately after the token is created.
+  - You can't change access (**read/write**) permissions for an API token after it's created.
+  - Tokens stop working when the user who created the token is deleted.
+
+We recommend the following for managing your tokens:
+- Create a generic user to create and manage tokens for writing data.
+- Store your tokens in a secure password vault for future access.
+{{% /cloud-only %}}
 {{% /note %}}
 
+- [Create a token in the InfluxDB UI](#create-a-token-in-the-influxdb-ui)
+- [Create a token using the influx CLI](#create-a-token-using-the-influx-cli)
+- [Create a token using the InfluxDB API](#create-a-token-using-the-influxdb-api)
+
 ## Create a token in the InfluxDB UI
 
+{{% oss-only %}}
+
 1. In the navigation menu on the left, select **Data (Load Data)** > **Tokens**.
 
     {{< nav-icon "load-data" >}}
 
 2. Click **{{< icon "plus" >}} Generate** and select a token type
-   (**Read/Write Token** or **All-Access Token**).
+   (**Read/Write Token** or **All Access API Token**).
 3. In the window that appears, enter a description for your token in the **Description** field.
 4. If generating a **read/write token**:
     - Search for and select buckets to read from in the **Read** pane.
     - Search for and select buckets to write to in the **Write** pane.
 5. Click **Save**.
 
+{{% /oss-only %}}
+
+
+{{% cloud-only %}}
+
+### Create an All-Access token
+
+1. In the navigation menu on the left, select **Load Data** > **Tokens**.
+
+    {{< nav-icon "load-data" >}}
+
+2. Click the **{{< icon "plus" >}} {{< caps >}}Generate API Token{{< /caps >}}** button.
+3. Select **All Access API Token**.
+
+### Create a Custom token
+
+1. In the navigation menu on the left, select **Load Data** > **Tokens**.
+
+    {{< nav-icon "load-data" >}}
+
+2. Click the **{{< icon "plus" >}} {{< caps >}}Generate API Token{{< /caps >}}** button.
+3. Select **Custom API Token**.
+4. When the **Generate a Personal API Token** window appears, enter a description. If you don't provide a description for the token, InfluxDB will generate a description from the permissions you assign.
+   For example, if you select **Read** for a bucket named "_monitoring" and **Write** for a bucket named "_tasks", InfluxDB will generate the description "Read buckets _monitoring Write buckets _tasks".
```

Comment:
Escape the underscores to prevent markdown from parsing them as `<em>`s.

```suggestion
   For example, if you select **Read** for a bucket named "\_monitoring" and **Write** for a bucket named "\_tasks", InfluxDB will generate the description "Read buckets \_monitoring Write buckets \_tasks".
```

Reply:
Thanks